### DISCUSSION TITLE
Sıralamada kaçıncı olduğunuzu öğrenmek için linke tıklayın

### DISCUSSION THREAD

**Discussion by Abdullahoda** [IBAX-io/go-ibax#623] [Ideas]
https://ibax.io/developer/honor-developers

**Comment by Akrepkapo** [comment]
Kaçmak derken kankam

**Reply by Abdullahoda** [reply]
😀

**Comment by Eemreecan** [comment]
iyiymiş, puan toplamanın farklı yolları varmı ki 30kya yakın yapanlar nasıl yapmış örneğin bilen aydınlatırsa sevinirim

**Reply by Abdullahoda** [reply]
Çekilişlerden kazanıyorlar bide hazım hatası yada proje sunuyorlar..

**Reply by Abdullahoda** [reply]
....
